File: src/test/fixtures/fsharp/opaqueBoolean.fs
module OpaqueBoolean

let flaggedPositionalBoolean () =
    configure true

let flaggedPositionalBooleanAmongOthers () =
    process 1 false

let suppressedNamedArgument () =
    configure(retries = true)

let suppressedNonCallUsage () =
    let ok = true
    ok

File: src/test/fixtures/csharp/OpaqueBoolean.cs
class OpaqueBooleans
{
    static void FlaggedPositionalBoolean() => Configure(true);
    static void FlaggedPositionalBooleanAmongOthers() => Process(1, false);
    static bool SuppressedNonCallUsage() => true;
    static void Configure(bool value) { }
    static void Process(int value, bool enabled) { }
}
